Tutorials/Creating-a-Custom-Dashboard/index.md

```diff
@@ -525,4 +541,8 @@ You can create your own custom angular services/resources, to interact with your
 
 ## The end
 
+With all of the steps completed, you should have a functional dashboard that will let the logged-in user see the changes they made!
+
+Please note this sample also includes document type changes, which is something editors do not usually do. As a result they might not be able to navigate to the document type by clicking the link on the dashboard.
```

Comment:
I'm not entirely sure I agree with the comments you added here, @Jay-umbr 🤔 

They will not see any Document Types if they do not have access to edit them.
I think instead, it would make sense to add a note like this further up in the article.
And then instead, have the note mention that the dashboard created in this tutorial is meant to show which content and media files the editors have been editing.

What do you think about that instead?